sdk/eventhub/event-hubs/package.json

```diff
@@ -110,7 +110,7 @@
     "@azure/core-amqp": "^3.1.0",
     "@azure/core-asynciterator-polyfill": "^1.0.0",
     "@azure/core-auth": "^1.3.0",
-    "@azure/core-tracing": "1.0.0-preview.13",
+    "@azure/core-tracing": "1.0.0-preview.14",
```

Comment:
> Haven't decided if this should be merged or if we should just publish to NPM under a custom tag in order to allow customers to dogfood. I know there's an internal IoT Hub engineer that is interested in helping out so we just need something for them to try.

Should this be a dev package instead for them to try? To avoid API changes or breaking anyone if we are still on the preview. @maorleger

Reply:
Yeah we will likely just publish a dev package from this branch and have a few dogfooders try that out